### PR TITLE
*: add Kubernetes templates

### PR DESCRIPTION
**Pull request by s-urbaniak**
/cc @lilic @paulfantom @simonpasquier @nikhita

I am not sure if I forgot something, please help me here :-)

@brancz @DirectXMan12 question: would you like to be added to the OWNERS/SECURITY_OWNERS files?

### CODE REVIEW

**Comment by s-urbaniak** [comment]
xref https://github.com/kubernetes/org/issues/2182

**Comment by paulfantom** [comment]
You can add [emeritus_approvers](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#emeritus) to OWNERS file to note who was previously working on the project.

File: OWNERS.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+approvers:
+  - prometheus-adapter-approvers
+
+reviewers:
+  - prometheus-adapter-approvers
+  - prometheus-adapter-reviewers
+
+emeritus_approvers:
+- brancz
```
Comment (brancz) [inline OWNERS.txt:14]: happy to continue to be owner/approver/reviewer, whatever makes your life easier :)

Comment (s-urbaniak) [reply OWNERS.txt:14]: awesome, thank you!

**Pull request merged**